explainers/subresource-loading.md

```diff
@@ -276,6 +277,15 @@ Instead of including a list of resources, the `<script>` defines a `scopes`.
 
 Any subresource under the `scopes` will be fetched from the bundle.
 
+## Contnent-Type and nosniff
```

Comment:
Maybe link to https://wpack-wg.github.io/bundled-responses/draft-ietf-wpack-bundled-responses.html#name-serving-constraints ?

Reply:
That's better! Thanks!
Mentioned the link, and removed duplicated explanation here.